src/views/InteractionDesigner.vue

```diff
@@ -260,6 +260,15 @@ export class InteractionDesigner extends mixins(Lang, Routes) {
     this.setActiveMainComponent({mainComponent: this.mainComponent})
   }
 
+  beforeMount(): void {
+    window.addEventListener('beforeunload', event => {
+      if (this.isEditable && this.hasFlowChanges) {
+        event.preventDefault()
+        event.returnValue = ''
```

Comment:
A returnValue with a string needs to be provided here (cannot use null). Previously, this string was used in the prompt, but that feature has since been deprecated (security reasons). So, an empty string is provided instead.